django/django_contracts/models/contract.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+
+from django.db import models, transaction
+from django.db.models import Manager
+from django.urls import reverse
+
+from django_contracts.compat import TimestampMixin
+from django_contracts.models.publishable import PublishableMixin, PublishStatus
+
+
+class LegalContract(TimestampMixin, PublishableMixin):
+    objects: Manager["LegalContract"]
+    versions: Manager["LegalContractVersion"]
+
+    title = models.CharField(max_length=128)
```

Comment:
Depending on how the field is intended to be used, 128 chars might be a bit on the low side?

Reply:
Perhaps, but I'd also be surprised if we go over it anytime within the next 2 years. I'd increase it if I made other changes but I don't think I'm going to.